framework/android/kodein-di-framework-android-x-viewmodel-savedstate/build.gradle.kts

```diff
@@ -0,0 +1,20 @@
+plugins {
+    id("org.kodein.library.android")
+}
+
+dependencies {
+    api(project(":framework:android:kodein-di-framework-android-x"))
+    implementation("androidx.core:core-ktx:1.6.0")
+    implementation("androidx.appcompat:appcompat:1.3.0")
+    implementation("androidx.activity:activity-ktx:1.2.3")
+    implementation("androidx.fragment:fragment-ktx:1.3.5")
+    val lifecycleVersion = "2.3.0"
+    implementation("androidx.lifecycle:lifecycle-viewmodel-ktx:$lifecycleVersion")
+    implementation("androidx.lifecycle:lifecycle-viewmodel-savedstate:$lifecycleVersion")
+    implementation("androidx.lifecycle:lifecycle-livedata-ktx:$lifecycleVersion")
```

Comment:
I'm not sure that this dependency is needed

Reply:
I will review this and tidy up or report back if there is an issue.